include/deal.II/sundials/arkode.h

```diff
@@ -634,9 +634,10 @@ namespace SUNDIALS
     get_arkode_memory() const;
 
     /**
-     * A function object that users need to supply and that is intended to
-     * reinit the given vector.
```

Comment:
I would keep the comment here.

Reply:
Isn't the info that you are not supposed to set this field enough?

Reply:
I would prefer if we still have the information about the original intention of this field.

Reply:
Ok, I changed it and also removed some outdated docmentation in the class.